src/libs/Clipboard/index.js

```diff
@@ -1,4 +1,34 @@
 // on Web/desktop this import will be replaced with `react-native-web`
+import _ from 'lodash';
```

Comment:
Use Single imports only to me it tree shakable. `import lodashGet from 'lodash/get'`.

Reply:
Also, the top line comment is now invalid.